.github/workflows/go-lint.yml

```diff
@@ -10,9 +10,26 @@ on:
 jobs:
   golangci:
     runs-on: ubuntu-latest
-
     steps:
       - uses: actions/checkout@v2
 
       - name: run lint
         run: make go-lint
+
+  gofmt:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: setup go
+        uses: actions/setup-go@v1
+        with:
+          go-version: '1.13'
+      - name: check gofmt
+        working-directory: ./
+        run: |
+          diffs=`gofmt -l .`
```

Comment:
IMO, `gofmt -l -d .` would be better.

Reply:
OK

Reply:
oh, no, I try this, it display too much info, it's unnecessary to show the detail in the ci.
@starsz  you could try for it..

Reply:
`-d display diffs instead of rewriting files`, so that we can solve the `go fmt` problem.